scripts/build/docker/Dockerfile.template

```diff
@@ -43,6 +47,10 @@ RUN apt-get update \
     zip \
     rpm
 
+# Workaround: Install a newer version of `tar` to make fpm in electron-builder work again
```

Comment:
IMHO these two 'Workaround' blocks should only be run in the `architecture == 'i686'` and `architecture == 'x86_64'` Dockerfiles - they're not needed in the `architecture == 'armv7hf'` Dockerfile as that uses a newer base-image and so the workaroud would actually _downgrade_ the version of `tar`.

Reply:
Something like 

```
<% if (architecture == 'i686' || architecture == 'x86_64') { %>
...
<% } %>
```

should do.

Reply:
Done